### DISCUSSION TITLE
Unravel_index operator missing the Boundary Check for the Input index

### DISCUSSION THREAD

**Discussion by Tommliu** [apache/incubator-mxnet#16771] [Q&A] [answered]
## Description
The current unravel_index method under tensor operator didn't check whether the input index is 
valid given the input shape.
### To Reproduce
![image](https://user-images.githubusercontent.com/40382964/68556864-e39f4100-046e-11ea-95a0-8a761436351b.png)

**Comment by lanking520** [comment]
Hi @Tommliu That seemed to be a numpy problem since you import numpy as np.

**Comment by Tommliu** [comment, marked as answer]
Should the unravel_index function in Mxnet follow the numpy convention?  I think a boundary check is necessary when using the unravel function.

**Reply by szha** [reply]
yes it should.
